src/Aspire.Dashboard/Otlp/Storage/TelemetryRepository.cs

```diff
@@ -266,14 +267,28 @@ public void AddLogsCore(AddContext context, OtlpApplication application, Repeate
         {
             foreach (var sl in scopeLogs)
             {
-                // Instrumentation Scope isn't commonly used for logs.
-                // Skip it for now until there is feedback that it has useful information.
+                OtlpScope? scope;
+                try
+                {
+                    var name = sl.Scope?.Name ?? string.Empty;
```

Comment:
What does missing scope mean?

Reply:
https://github.com/dotnet/aspire/blob/a3ed2b5d202f55179cb9c3985663c0995fb8e0a2/src/Aspire.Dashboard/Otlp/opentelemetry/proto/logs/v1/logs.proto#L65-L68

Reply:
Thanks for adding comment in the code too.